### ISSUE TITLE
Pigsteel armor trim icons displaying diamond trim

### ISSUE THREAD

**Issue by asdasdasadg** [Uraneptus/Pigsteel-forge-1.16#5]
![image](https://github.com/Uraneptus/Pigsteel-forge/assets/84078980/8223278c-c5ba-4d5b-b725-92a23f6a071a)

**Comment by Uraneptus** [comment]
Yeah that's cuz the trim system mojang setup is horrible sadly. Will probably use Trimmed library in the future to fix this
